README.md

```diff
@@ -1,92 +1,145 @@
-  - Disabling the default Cmd + Tab keybind, will allow a user to set a custom keybind
-      - User selects one of the modifiers presented on the screen
-      - User presses `Record Keybind` button
-      - User presses a singular key on the keyboard
-      - Keybind is now set! 
-
-      ![Set keybind](./resources/setKeybind.gif)
-- **How do I use the dock peeking functionality?**
-  - Simply hover over any application with active windows in the dock.
-- **What are the traffic light buttons that appear in the preview window?**
-  - 🟣 Quit the window's app. You can hold the Option (⌥) key while clicking to **force** quit.
-  - 🔴 Close the window
-  - 🟡 Minimize the window
-  - 🟢 Enter the window to full screen
-
-### FAQ
-
-- I disabled the menu bar icon and now I can't access settings
-  - When DockDoor initially opens, the menu icon will be visible for 10 seconds, until it disappears. This way, you can access the settings even with the icon disabled. Just relaunch the app and click it before it disappears.
-- I click on the purple quit button in the preview and the app doesn't close
-  - You can hold the Option (⌥) key while clicking to force quit.
- 
-## Installation (for contributors)
+### Dock Peek feature
+Hover over any application’s icon in the Dock to see its open windows.
+Each traffic light button you see serves an intuitive function.
+> [!TIP]
+> Hold **`Option`** (⌥) while clicking the **`Quit`** button to **force quit** the window’s application.
+<p align="center">
+  <img width="400" height="auto" src="./resources/dockPeekCommented.png">
+</p>
 
-### Prerequisites
+### Windows-like `Alt` + `Tab` window switcher
+With the default settings, hold down `Command` (⌘) and press `Tab` (⇥) repeatedly until the desired window is highlighted. To go back, press `Shift` (⇧) in addition to `Tab`.
+Release both keys to switch to the selected window.
 
-- macOS 14.0 or later.
-- Xcode installed on your machine.
+#### Customization
+You can customize the default keybind by disabling the option `Use default macOS keybind ⌘ + ⇥`.
+- Select one of the initialization keys (`Control`, `Option` or `Command`)
+- Click on `Start Recording Keybind`
+- Press a single key on your keyboard
+- Custom keybind is set!
+![Set keybind](./resources/setKeybind.gif)
 
-### Setting Up the Project
 
-1. Fork the repository.
-2. Clone your forked repository to your local machine.
-3. Open the project in Xcode.
-4. Build and run the project.
+## Frequently Asked Questions
 
-## How to Contribute
+### How to access DockDoor settings?
+If you disabled `Show Menu Bar Icon` in `General`, search for `DockDoor` using the built-in Spotlight or another searching app. Open DockDoor and the settings should open automatically.
 
-Contributions to DockDoor are welcome! Here’s how you can get started:
+If this option is enabled, click on *door* icon on the menu bar and select `Open Settings` in the contextual menu.
```

Comment:
should be:

click on the *door* icon in the menu bar

Reply:
Resolved in 7102ac8585dc9d363092754a4d74b9955e767d7d